website/docs/cloud-docs/api-docs/applies.mdx

```diff
@@ -174,3 +152,50 @@ _Using Terraform Cloud Agents_
   }
 }
 ```
+
+## Recover a failed state upload
+
+`GET /applies/:id/errored-state`
+
+| Parameter | Description                               |
+| --------- | ----------------------------------------- |
+| `id`      | The ID of the apply to recover state for. |
+
+It is possible that during the course of a Run, Terraform may fail to upload a
+state file. This can happen for a variety of reasons, but should be an
+exceptionally rare occurance. Terraform Cloud Agent versions greater than 1.12.0
+include a fallback mechanism which attempts to upload the state directly to
+Terraform Cloud's backend storage system in these cases. This endpoint then
+makes the raw data from these failed uploads available to users who are
+authorized to read the state contents.
```

Comment:
```suggestion
It is possible that during the course of a Run, Terraform may fail to upload a
state file. This endpoint makes the raw data from these failed uploads available to users who are
authorized to read the state contents. These errors can happen for a variety of reasons, but should be an
exceptionally rare occurrence. Terraform Cloud Agent versions greater than 1.12.0
include a fallback mechanism which attempts to upload the state directly to
Terraform Cloud's backend storage system in these cases. 
```

Reply:
I found this a bit harder to read personally, because it switches from talking about the condition (Terraform failing to upload a state file) to talking about the endpoint, and then back again.